packages/app/src/templates/components/templates/templates.component.tsx

```diff
@@ -29,6 +76,17 @@ export const Templates: FunctionComponent<TemplatesProps> = ({
     setNewTemplateOpenState(false)
   }
 
+  const handleCloseDeleteModal = () => {
+    setDeleteModalOpen(false)
+  }
+  const handleOpenDeleteModal = (ids: string[]) => {
+    setDeleteModalOpen(true)
+    setDeletedTemplates(ids)
+  }
+  const handleDeleteButton = () => {
+    deleteTemplates(deletedTemplates)
```

Comment:
Not 100% sure, but shouldn't it be `await deleteTemplates(deletedTemplates)`?

Reply:
Added in [4ed3a46](https://github.com/mudita/mudita-center/pull/764/commits/4ed3a46e4c6351403608986075954b248f669eab)